test/test_positioning.rb

```diff
@@ -501,10 +487,10 @@ def test_lock_positioning_scope_with_only_scope_columns_on_persisted_record_and_
   end
 
   def test_lock_positioning_scope_without_scope_association
-    category = Category.create name: "Category"
-    mechanisms = Positioning::Mechanisms.new(category, :position)
+    product = Product.create name: "Product"
```

Comment:
All I did in this file besides extraction of healing tests, is replaces categories with products, as categories became a tree.